source/guides/infrastructure/azure/cip.html.md.erb

```diff
@@ -0,0 +1,78 @@
+---
+title: Cloud Infrastructure Platform
+weight: 10
+last_reviewed_on: 2021-11-30
+review_in: 1 year
+hide_in_navigation: true
+layout: navigation
+link_in_toc: true
+summary: CIP is the default platform in DfE based on Azure.
+---
+
+It is based on Azure and provides preconfigured subscriptions including security guardrails.
+
+It provides access to most of Azure resources including app services, container instances, virtual networks, managed databases, front door, key vault, storage accounts…
+
+It can be used for any workload, especially ones which don’t fit on Gov.uk paas. It can also be used alongsite paas to complement it. For example: to store secrets, terraform state, backups…
+
+Portal: [https://portal.azure.com/](https://portal.azure.com/)
+
+## Documentation
+* [CIP platform docs](https://docs.platform.education.gov.uk/index.html)
+
+## Onboarding users
+Use the service portal (service now) to create the onboarding request:
+
+1. Request something
+1. CIP Request
```

Comment:
Could these steps be a bit more explicit? (eg "Select 'CIP Request' from the dropdown")